commands/registry.go

```diff
@@ -0,0 +1,244 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/doctl/do"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+	k8sapiv1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sjson "k8s.io/apimachinery/pkg/runtime/serializer/json"
+)
+
+type dockerConfig struct {
+	Auths map[string]struct {
+		Auth string `json:"auth,omitempty"`
+	} `json:"auths"`
+}
+
+const registryHostname = "registry.digitalocean.com"
+
+// Registry creates the registry command
+func Registry() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "registry",
+			Aliases: []string{"reg", "r"},
+			Short:   "[beta] registry commands",
+			Long:    "[beta] registry is used to access container registry commands",
+			Hidden:  true,
+		},
+	}
+
+	CmdBuilder(cmd, RunRegistryCreate, "create <registry-name>", "create container registry", Writer)
+
+	CmdBuilder(cmd, RunRegistryGet, "get", "get the container registry", Writer, aliasOpt("g"), displayerType(&displayers.Registry{}))
+
+	cmdRunRegistryDelete := CmdBuilder(cmd, RunRegistryDelete, "delete", "delete the container registry", Writer, aliasOpt("del"))
```

Comment:
nit: I think we need to add more aliases here as most command stick to `aliasOpt("d", "del", "rm")`

I think get being `g` is correct.

Reply:
👍🏼

Reply:
👍🏼